deps/npm/node_modules/socks/typings/common/receivebuffer.d.ts

```diff
@@ -1,12 +0,0 @@
-/// <reference types="node" />
-declare class ReceiveBuffer {
```

Comment:
This change seems unrelated

Reply:
Hmmm...yeah, no idea how that happened. Probably some rebasing weirdness as I also did a rebase.

Reply:
Fixed.